specification/desktopvirtualization/suppressions.yaml

```diff
@@ -0,0 +1,4 @@
+- tool: AvoidAdditionalProperties
+  paths:
+    - resource-manager/Microsoft.DesktopVirtualization/preview/2024-08-08-preview/*.json
```

Comment:
>ew/2024-08-08-preview/*.json [](http://example.com/codeflow?start=60&length=28)

this is too generic and ARM is trying to reduce instances of unschematized APIs to make it easier for consumers of those APIs to understand them and use them reliably, not to mention provide better support in the various SDKs.

your PR does not add new additional properties why are you ading this suppresion file for any possible file?

Reply:
The suppression only targets desktopVirtualization.json because that is the only json file in the directory, I will update the suppression to use "desktopVirtualization.json" rather than *.json.

As far as why I am adding the suppression file even though my PR does not add any additional properties. The LintDiff check for AvoidAdditionalProperties has been incorrectly giving an error ever since it was added, we already had the vmTags property in our Api. in a previous review Roopesh suggested that I add the suppression for each individual version so we don't miss future issues.

Reply:
@razvanbadea-msft can you add the approved-suppression label to the PR?